### ISSUE TITLE
support rust's wasm-pack  *.d.ts?

### ISSUE THREAD

**Issue by sonygod** [haxiomic/dts2hx#121]
support rust's wasm-pack  *.d.ts?

I try to use this cmd 

   `npx dts2hx pkg/index.d.ts  --tsconfig .\tsconfig.json`

**> Converting module pkg/index
> Error: TypeError: Cannot read properties of undefined (reading 'length')**

here is the test file.

```ts
/* tslint:disable */
/* eslint-disable */
/**
* @param {Float32Array} v0
* @param {Float32Array} v1
* @param {Float32Array} v2
* @param {Float32Array} u0
* @param {Float32Array} u1
* @param {Float32Array} u2
* @param {number | undefined} epsilon
* @returns {boolean}
*/
export function noDivTriTriIsect(v0: Float32Array, v1: Float32Array, v2: Float32Array, u0: Float32Array, u1: Float32Array, u2: Float32Array, epsilon?: number): boolean;
/**
* @param {Float32Array} m1
* @param {Float32Array} m2
* @param {number | undefined} epsilon
* @returns {boolean}
*/
export function meshMeshIsect(m1: Float32Array, m2: Float32Array, epsilon?: number): boolean;

export type InitInput = RequestInfo | URL | Response | BufferSource | WebAssembly.Module;

export interface InitOutput {
  readonly memory: WebAssembly.Memory;
  readonly noDivTriTriIsect: (a: number, b: number, c: number, d: number, e: number, f: number, g: number, h: number, i: number, j: number, k: number, l: number, m: number, n: number) => number;
  readonly meshMeshIsect: (a: number, b: number, c: number, d: number, e: number, f: number) => number;
  readonly __wbindgen_malloc: (a: number) => number;
}

export type SyncInitInput = BufferSource | WebAssembly.Module;
/**
* Instantiates the given `module`, which can either be bytes or
* a precompiled `WebAssembly.Module`.
*
* @param {SyncInitInput} module
*
* @returns {InitOutput}
*/
export function initSync(module: SyncInitInput): InitOutput;

/**
* If `module_or_path` is {RequestInfo} or {URL}, makes a request and
* for everything else, calls `WebAssembly.instantiate` directly.
*
* @param {InitInput | Promise<InitInput>} module_or_path
*
* @returns {Promise<InitOutput>}
*/
export default function init (module_or_path?: InitInput | Promise<InitInput>): Promise<InitOutput>;

```

**Comment by haxiomic** [comment]
Works fine for me but I don’t have your tsconfig.json, could you send that over too?

**Comment by sonygod** [comment]
actually,this file was generated  by WASM-PACK tool and default have no tsconfig.json, this files was copy from other 

project. 

```json
{
	"compilerOptions": {
		"target": "es2016",
		"module": "commonjs",
		"esModuleInterop": true,
	"forceConsistentCasingInFileNames": true,
		"strict": true,
		"skipLibCheck": true,
		"baseUrl": ".",
		"noEmit": true,
		"noUnusedParameters": true,
		"strictNullChecks": true
	},
	"include": ["pkg","src"],
	"exclude": ["node_modules"]
}
```

<img width="476" alt="image" src="https://user-images.githubusercontent.com/748320/193165521-1cf93664-6581-4c89-a77a-b0b9d288329e.png">

**Comment by haxiomic** [comment]
<img width="750" alt="Screenshot 2022-10-01 at 10 43 41" src="https://user-images.githubusercontent.com/3742992/193401147-9468bbf4-05e6-4acf-9ff7-402ca77818c2.png">
Works for me testing on macOS, no errors and the externs look correct

I'll test on windows later in the week

**Comment by haxiomic** [comment]
Should be fixed in 0.19.2, let me know :) f0fdfa20128f4aab87cad18bfc0c31b2279af4ad
